.ci/updatecli/values.yml

```diff
@@ -0,0 +1,4 @@
+github:
+  owner: "elastic"
+  repository: "cloudbeat"
+  branch: "main"
```

Comment:
Can you please add trailing whitespace here so that git doesn't complain?

Reply:
do you mean an EOF?